CHANGELOG.md

```diff
@@ -17,6 +17,16 @@
     cart. You can set it through the new `Spree::Config.variant_pricer_class` setting. This
     class also knows which `PricingOptions` class it cooperates with.
 
+    #### Order Merging on Sign In:
+
+    Previously a before filter added in `core/lib/spree/core/controller_helpers/order.rb`
+    would cause sql queries to be used on almost every request in the frontend.
+    If you do not use Solidus Auth you will need to hook into this helper and
+    call `set_current_order` where your user signs in. This merges incomplete
+    orders a user has going with their current cart.  If you do use
+    Solidus Auth you will need to make sure you use a current enough
+    version that includes this explicit call.
```

Comment:
I could change my entry to instruct them they could also just emulate the `set_current_order` logic themselves instead of hooking into the helper and using it if you would like, but I still don't see where Solidus Frontend comes into play.  The merging logic used to be called there but this is removing that.